plugins/com.aptana.git.ui/src/com/aptana/git/ui/internal/history/GitCompareFileRevisionEditorInput.java

```diff
@@ -399,12 +401,12 @@ private String getContentIdentifier(ITypedElement element)
 				try
 				{
 					IStorage storage = ((LocalFileRevision) fileObject).getStorage(new NullProgressMonitor());
-					if (Utils.getAdapter(storage, IFileState.class) != null)
```

Comment:
Correct way to use this - 
Adapters.adapt(storage, IFileState.class);

Reply:
Good one again. I haven't been able to find this out. Thank you! 👍